docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3.4'
```

Comment:
this must be reverted to 3.7

Reply:
ok

Reply:
The problem is I'm using a Ubuntu 18 Virtual Machine (installed to work at home due to COVID19) that do not support version 3.7. When I return in office I can change it